jquery.simulate.js

```diff
@@ -200,8 +200,8 @@ $.extend( $.simulate.prototype, {
 		return event;
 	},
 
-	dispatchEvent: function( elem, type, event ) {
-		if ( elem[ type ] ) {
+	dispatchEvent: function( elem, type, event, options ) {
+		if ( elem[ type ] && !options ) {
```

Comment:
It was added in 206bbf873e8f63646fcd1daa3c4713c2ebd66196 but that seems way too broad for the bug it's trying to fix. For example it's not passing the event object. The current case will run not only `click` but also `focus` and `blur` which don't seem to need the workaround. I think this *may* also be just an older IE problem (the YUI ticket link is dead now but gh-1 says it is) so it might be good to isolate this case via a feature detect and not make other browsers do it.